src/decanter/core/extra/utils.py

```diff
@@ -14,6 +14,7 @@ def check_response(response, key=None):
     """
     code = response.status_code
     if not 200 <= code < 300:
+        print('response: ' + str(response))
```

Comment:
use logging instead of print

Reply:
This was used for debugging. I just removed it!